staff/blas-toth/challenges/calculadora/calculate.js

```diff
@@ -0,0 +1,13 @@
+function calculate(number1, operation, number2) {
+    // TODO implement the four operations + - / *
+
+    if (operation === '+') {
+        result = Number(auxiliar) + Number(memory)
```

Comment:
add semicolon in all statements where you assign values (=)

Reply:
Thank you, done!